src/managers/KeybindManager.cpp

```diff
@@ -2216,10 +2216,12 @@ SDispatchResult CKeybindManager::circleNext(std::string arg) {
     else if (args.contains("float") || args.contains("floating"))
         floatStatus = true;
 
-    if (args.contains("prev") || args.contains("p") || args.contains("last") || args.contains("l"))
-        switchToWindow(g_pCompositor->getPrevWindowOnWorkspace(g_pCompositor->m_pLastWindow.lock(), true, floatStatus));
-    else
-        switchToWindow(g_pCompositor->getNextWindowOnWorkspace(g_pCompositor->m_pLastWindow.lock(), true, floatStatus));
+    const auto  VISIBLE = args.contains("visible") || args.contains("v");
+    const auto& w       = (args.contains("prev") || args.contains("p") || args.contains("last") || args.contains("l")) ?
```

Comment:
`contains("l")` will always be true for `visible`, unintended?

Reply:
that's not true... args is a vector of strings. `contains` only looks for an exact match in that vector, without doing any substring searching

Reply:
woops, my bad.